src/Microsoft.Fx.Portability/ObjectModel/CloudApiCatalogLookup.cs

```diff
@@ -211,6 +213,8 @@ public IEnumerable<string> GetAncestors(string docId)
             }
         }
 
+        public DateTimeOffset BuiltOn { get { return _builtOn; } }
```

Comment:
The name "BuiltOn" seems a little ambiguous - built on what? How about 'LastModified' or something?